cmd/storagenode-updater/main_test.go

```diff
@@ -192,10 +193,19 @@ func testVersionControlWithUpdates(ctx *testcontext.Context, t *testing.T, updat
 
 	ts := httptest.NewServer(&mux)
 
+	var randSeed version.RolloutBytes
+	_, err := rand.Read(randSeed[:])
```

Comment:
nit: you can think about moving this to `internal/testrand` package

Reply:
nice - i was not aware of that, thanks!

Reply:
nice - i was not aware of that, thanks!